lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -175,6 +174,10 @@ describe('ContentMetadataComponent', () => {
         nodesApiService = TestBed.inject(NodesApiService);
         tagService = TestBed.inject(TagService);
         categoryService = TestBed.inject(CategoryService);
+        notificationService = TestBed.inject(NotificationService);
+        const propertyDescriptorsService = TestBed.inject(PropertyDescriptorsService);
+        const classesApi = propertyDescriptorsService['classesApi'];
+        changeDetectorRef = fixture.componentRef.injector.get(ChangeDetectorRef);
```

Comment:
Why you take reference to ChangeDetectorRef by fixture.componentRef.injector.get(ChangeDetectorRef)? 
You can just call fixture.detectChanges(). You don't need that whole line and reference to ChangeDetectorRef.

Reply:
done